### ISSUE TITLE
UserAccount Mail Validation + Deeplink

### ISSUE THREAD

**Issue by mlecoq** [DispatchMe/meteor-deep-link#5]
Hi,

I would like to open links from email validation in my app.

I have added the following cordova plugin 

https://github.com/yoik/cordova-yoik-intent-filter

with URL_SCHEME https and HOST_NAME myhost.com

DeepLink handles my urls (from my validation email):

```
  DeepLink.on('https', (data, url, scheme, path) => {
    FlowRouter.go(path);
  });
```

My problem is the path parameter: it is filled with 'myhost'

**Comment by mlecoq** [comment]
I have made a pull request 

https://github.com/DispatchMe/meteor-deep-link/pull/6

**Comment by bmanturner** [comment]
@raix This would be incredibly useful for my current project. Can we please merge it in or do I need to fork it?

**Comment by raix** [comment]
Maybe @aldeed can help

**Comment by aldeed** [comment]
@bmanturner Merged and released in 4.1.0
